lib/grape/validations/types/custom_type_coercer.rb

```diff
@@ -162,15 +162,15 @@ def enforce_symbolized_keys(type, method)
             lambda do |val|
               method.call(val).tap do |new_value|
                 new_value.each do |item|
-                  Hashie.symbolize_keys!(item) if item.is_a? Hash
```

Comment:
@dblock This slipped through the net. From the description above this method is supposed to wrap the result of the method and transform any Hash to be symbolized, that's fine, I can update. There is not direct test of this method, but reading the code it looks to me like the line above should not be iterating with each but map to construct a new Array or Set;

```ruby
                new_value.map do |item|
```

Correct?